src/helper/externalApiRequest.js

```diff
@@ -0,0 +1,12 @@
+const request = require('request-promise-native');
+const { REQUEST_TIMEOUT } = require('../../config/globals');
+
+const externalApiRequest = (baseUrl) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4499680)